desc/compute/_equil.py

```diff
@@ -562,6 +562,26 @@ def _e_sup_helical(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="e^helical*sqrt(g)",
+    label=" \\sqrt{g}(B^{\\theta} \\nabla \\zeta - B^{\\zeta} \\nabla \\theta)",
+    units="T \\cdot m^{2}",
+    units_long="Tesla * square meter",
+    description="Helical basis vector weighted by 3-D volume Jacobian",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["B^theta", "B^zeta", "e^theta*sqrt(g)", "e^zeta"],
+)
+def _e_sup_helical_times_sqrt_g(params, transforms, profiles, data, **kwargs):
+    data["e^helical*sqrt(g)"] = (
+        data["B^zeta"] * data["e^theta*sqrt(g)"].T - data["B^theta"] * data["e^zeta"].T
```

Comment:
Don't forget to multiply `this_term = data["B^theta"] * data["e^zeta"].T` by Jacobian. `this_term * sqrt(g)` will be computed as zero at magnetic axis, but you want it to be valid everywhere. Then it's good to merge.

Might be good to `git cherry-pick eb09288fc54c31a08a3d51630819ec80b9dca071` so that eb09288fc54c31a08a3d51630819ec80b9dca071 is merged in this PR.

Reply:
ah good point thanks